web/pom.xml

```diff
@@ -290,7 +290,7 @@
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-java-sdk</artifactId>
-            <version>1.12.377</version>
```

Comment:
why we downgrading the version?

Reply:
There was an issue previously where getting the data from the bucket would not work with the version we had if you recall

Reply:
Actually I tested it out with 1.12 and cannot recreate the error so I will update it back to 1.12